test/test_Bassa_endpoint.py

```diff
@@ -0,0 +1,28 @@
+import unittest
+import requests
+
+headers = {
+'Host': 'localhost:5000',
+'Accept': 'application/json, text/plain, */*',
+'Accept-Language': 'en-US,en;q=0.5',
+'Accept-Encoding': 'gzip, deflate',
+'Content-Type': 'application/x-www-form-urlencoded',
+'Origin': 'http://localhost:3000'
+}
+correct_username="rand"
+correct_password="pass"
+incorrect_username="admin"
+incorrect_password="admin"
+correct_string="user_name="+correct_username+"&password="+correct_password
+incorrect_string="user_name="+incorrect_username+"&password="+incorrect_password
+payload = {''}
+class TestFlaskAPIUsingRequests(unittest.TestCase):
+    def test_api_login_returns_auth_level(self):
+        resp = requests.post('http://localhost:5000/api/login',correct_string,headers=headers)
+        def check_valid_auth(self):
```

Comment:
Why do we have a function inside a function?

Reply:
i wanted to seperate the login auth as another function just like you asked

Reply:
How else do I separate the check auth function as the response is received only after the POST...

Reply:
btw i have submitted my task for review as well ([Link](https://codein.withgoogle.com/dashboard/task-instances/5836999971831808/)) so you could review that as well

Reply:
also could you please ping me in gitter when you come online

Reply:
Ah Cool. Shall we have proper indentation for the following line

Reply:
@malithsen i see that you are online .. could you come on gitter as  i have a few queries regarding this project